### PR TITLE
Add some missing things to dom.nim

### PR DESCRIPTION
**Pull request by treeform**
Add window.performance, window.performance.now(), window.devicePixelRatio, window.scrollX and Y, style.overflowX and Y, node.getBoundingRect.

### CODE REVIEW

File: lib/js/dom.nim [hunk 1]
```diff
@@ -400,6 +406,40 @@ type
...
+    redirectEnd*: float
+    redirectStart*: float
+    requestStart*: float
+    responseEnd*: float
+    responseStart*: float
+    secureConnectionStart*: float
+    unloadEventEnd*: float
+    unloadEventStart*: float
+
+  Performance* {.importc.} = object
```
Comment (dom96) [inline lib/js/dom.nim]: Shouldn't all these be ``ref``s?

Comment (treeform) [reply lib/js/dom.nim]: Yes you are right. I guess they work without being refs..., but if some one wants to create their own they might have issues.

**Comment by treeform** [comment]
I think continuous-integration/travis-ci/pr is stuck, its been waiting to start for many hours.

**Comment by dom96** [comment]
oh, another thing is your PR is against ``master``, you'll need to create a new one against ``devel``.

**Pull request closed without merging**